app/poi_api/api.py

```diff
@@ -1,32 +1,32 @@
-from .poi_service import POI_Service
+from .service import list_POI, get_details_for_POI, new_POI_suggestion
 from flask import jsonify
-from .poi_errors import POINotFoundError, InvalidPOISuggestionError
-
-poi_service_class = POI_Service()
+from .errors import POINotFoundError, InvalidPOISuggestionError
+from typing import Dict
 
 # TODO Exception handling
 # TODO type hints
 def get_all_POI():
     try:
-        list_all_poi = poi_service_class.get_all_POI()
+        list_all_poi = list_POI()
         return jsonify([poi.to_dict() for poi in list_all_poi]), 200
     except Exception as e:
         return {"error": str(e)}, 500
 
 
-def get_POI(poi_id):
+def get_POI(poi_id: str):
     try:
-        get_poi = poi_service_class.get_POI(poi_id)
+        get_poi = get_details_for_POI(poi_id)
         return jsonify(get_poi.to_dict()), 200
     except POINotFoundError as e:
         return {"error": str(e)}, 404
     except Exception as e:
         return {"error": str(e)}, 500
 
 
-def suggest_new_POI(poi_suggestion):

```

Comment:
You should be using the with_auth_user decorator for this function to get the user sending the request, instead of having them put their identity in the request body.